src/ci/docker/host-x86_64/mingw-check-1/Dockerfile

```diff
@@ -48,7 +48,7 @@ ENV SCRIPT \
            python3 ../x.py build src/tools/build-manifest && \
            python3 ../x.py test --stage 0 src/tools/compiletest && \
            python3 ../x.py check compiletest --set build.compiletest-use-stage0-libtest=true && \
-           python3 ../x.py check --stage 1 --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu && \
+           python3 ../x.py check --stage 2 --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu && \
```

Comment:
Suggestion: actually, can you add a comment for this specifically that we're exercising *cross-compile* here, so `--stage 1` isn't sufficient?